sample/F8/tracks

```diff
@@ -0,0 +1,209 @@
+[{
+		"id": 1,
+		"name": "ADS & MONETIZATION",
```

Comment:
![screenshot_20170605_154934](https://cloud.githubusercontent.com/assets/3874064/26780278/9c5ca4d8-4a06-11e7-9127-8fa23626d4dc.png)

You can see the difference in quotes here